zsh/configs/post/path.zsh

```diff
@@ -1,9 +1,11 @@
 # ensure dotfiles bin directory is loaded first
 PATH="$HOME/.bin:/usr/local/sbin:$PATH"
 
-# load ASDF, falling back to rbenv if not available
-if [ -d "$HOME/.asdf" ]; then
+# load ASDF, either from home dir or brew install
+if [ -f "$HOME/.asdf/asdf.sh" ]; then
   . $HOME/.asdf/asdf.sh
+elif ASDF_PREFIX=$(brew --prefix asdf); then
```

Comment:
Hey @danbee, thanks for opening this PR! I just got a new machine and installed `asdf` with Homebrew, so I've been running into this issue myself.

What do you think about the following to address the concerns raised by Mike and Eebs?

```bash
# load ASDF, falling back to rbenv if not available
if [ -f "$HOME/.asdf/asdf.sh" ]; then
  . $HOME/.asdf/asdf.sh
elif command -v brew > /dev/null; then
  ASDF_EXECUTABLE="$(brew --prefix asdf)/asdf.sh"

  if [ -f "$ASDF_EXECUTABLE" ]; then
    . "$ASDF_EXECUTABLE"
  fi
fi
```

Reply:
Or, much more simply

```zsh
# load ASDF, falling back to rbenv if not available
if [ -f "$HOME/.asdf/asdf.sh" ]; then
  . $HOME/.asdf/asdf.sh
elif [ -f "/usr/local/opts/asdf/asdf.sh" ]; then
  . "/usr/local/opts/asdf/asdf.sh"
fi
```

since `brew --prefix` is agonizingly slow and for the vast majority of Homebrew installs the prefix will be `/usr/local`.